### PR TITLE
Set version as 23.1.3 and minor changelog update

### PR DESCRIPTION
**Pull request by siladu**

### CODE REVIEW

File: ethereum/referencetests/src/reference-test/external-resources
Comment (jframe) [inline ethereum/referencetests/src/reference-test/external-resources:1]: Did you mean to change this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 ## 23.1.3
```
Comment (jframe) [inline CHANGELOG.md:4]: We should add a note that this is a hotfix for the nimbus issue

File: ethereum/referencetests/src/reference-test/external-resources
Comment (siladu) [reply ethereum/referencetests/src/reference-test/external-resources:1]: that guy again! good spot

Comment (siladu) [reply ethereum/referencetests/src/reference-test/external-resources:1]: ah, actually it's fine I think. It'll have been wrong on the release branch

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**